helix-view/src/theme.rs

```diff
@@ -72,8 +90,8 @@ impl Loader {
 
     /// Lists all theme names available in default and user directory
     pub fn names(&self) -> Vec<String> {
```

Comment:
Actually I don't know why this exists, no using reference for this method.

Reply:
I think this was supposed to be used by the theme completion